Dockerfile

```diff
@@ -134,6 +135,8 @@ RUN sed -i -e 's/include_try \/usr\/share\/dovecot\/protocols\.d/include_try \/e
   # create directory for certificates created by mkcert
   mkdir /etc/dovecot/ssl && \
   chmod 755 /etc/dovecot/ssl  && \
+  touch /etc/dovecot/ssl/.rand && \
+  sed -i 's/dev\/urandom/etc\/dovecot\/ssl\/\.rand/' /usr/share/dovecot/dovecot-openssl.cnf && \
```

Comment:
Why are you changing the dovecot random function in this way and how is it related to /etc/aliases? If this changes should be done (and please explain why) it should probably go in a separate commit.

Reply:
This is an unrelated build fix that I tossed in quickly.  As I recall, during the build phase, openssl throws an error when generating the dovecot key.  IIRC it's because it tries to write entropy to  /dev/urandom which of course is not allowed.  So now I have it creating an empty, writable file for it to put its entropy in, and that filename gets put in dovecot-openssl.cnf.

Reply:
That is the build message:

    Generating a RSA private key
    ....+++++
    ........+++++
    writing new private key to '/etc/dovecot/ssl/dovecot.key'
    -----
    Cannot write random bytes:
    140123681975557:error:2407007A:random number generator:RAND_write_file:Not a regular file:../crypto/rand/randfile.c:183:Filename=/dev/urandom

If I understand https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=922732#20 correctly, it should be safe to remove RANDFILE completely from the configuration:

    | OpenSSL 1.1.1 introduced a new random generator (CSPRNG) with an improved
    | seeding mechanism. The new seeding mechanism makes it unnecessary to
    | define a RANDFILE for saving and restoring randomness. This option is
    | retained mainly for compatibility reasons.

In any case, this should be discussed/done in a separate PR.

Reply:
@casperklein I'm completely with you there.

@MakerMatrix when we have finished the migration to https://github.com/docker-mailserver/docker-mailserver, could open a few several small PRs that go through each of your changes one by one?

Reply:
@casperklein @MakerMatrix I'm currently working on a bigger improvement of the `Dockerfile`. @casperklein How would I go about removing the RANDFILE completely?

Reply:
@casperklein Nevermind - I did it by removing `RANDFILE` from `mkcert.sh`. Done.

Reply:
> @MakerMatrix when we have finished the migration to https://github.com/docker-mailserver/docker-mailserver, could open a few several small PRs that go through each of your changes one by one?

Happy to.